code/datums/components/food/edible.dm

```diff
@@ -65,6 +79,29 @@ Behavior that's still missing from this component that original food items had t
 			else
 				owner.reagents.add_reagent(rid, amount)
 
+/datum/component/edible/InheritComponent(datum/component/C,
+	i_am_original,
+	list/initial_reagents,
+	food_flags = NONE,
+	foodtypes = NONE,
+	volume = 50,
+	eat_time = 30,
+	list/tastes,
+	list/eatverbs = list("bite","chew","nibble","gnaw","gobble","chomp"),
+	bite_consumption = 2,
+	datum/callback/after_eat,
+	datum/callback/on_consume)
+
+	. = ..()
+	src.bite_consumption = bite_consumption
+	src.food_flags = food_flags
+	src.foodtypes = foodtypes
+	src.eat_time = eat_time
+	src.eatverbs = eatverbs
+	src.junkiness = junkiness
+	src.after_eat = after_eat
+	src.on_consume = on_consume
```

Comment:
src is implied

Reply:
on_consume = on_consume is not explicit ;)